steps/utility/utility.go

```diff
@@ -42,8 +42,7 @@ var _ = gauge.Step("Switch to project <projectName>", func(projectName string) {
 	gauge.GetScenarioStore()["namespace"] = projectName
 })
 
-var _ = gauge.Step("Validate route url for pipelines tutorial", func() {
-	expectedOutput := "Cat 🐺 vs Dog 🐶"
+var _ = gauge.Step("Validate route url which contains <expectedOutput>", func(expectedOutput string) {
```

Comment:
Please rename the step so that it reads more naturally... `Validate that route URL contains <expectedOutput>`

Reply:
Done